src/Quiz.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import './App.css';
+import Demon_sprite from './data/spr/Demon_sprite.png'
+import Imp_sprite from './data/spr/Imp_sprite.png'
+import useSound from 'use-sound';
+import dssgtatk from './data/snd/dssgtatk.wav';
+// /static/media/dssgtatk.cd1a3ad1.wav
+
+function Quiz(props) {
+  // const props = {
+  //   image: "pathToPic",
+  //   sounds: [0, 1, 2]
+  // }
+
+  const [selectedButton, setSelectedButton] = useState(0);
+  const [isAnswered, setIsAnswered] = useState(false);
+  const [isDisabledButtons, setIsDisabledButtons] = useState(false);
+  const [hurlAcceleration, setHurlAcceleration] = useState(0);
+
+  const soundUrl1 = props.sounds[0];
```

Comment:
`soundUrl1` const used only once, hence you could avoid storing `props.sounds[0]` in const but use it directly.

Reply:
Yeah, fixed.